nni/algorithms/compression/pytorch/quantization/quantizers.py

```diff
@@ -344,9 +367,39 @@ def export_model(self, model_path, calibration_path=None, onnx_path=None, input_
 
         return calibration_config
 
-    def fold_bn(self, config, **kwargs):
-        # TODO simulate folded weight
-        pass
+    def fold_bn(self, *inputs, wrapper):
```

Comment:
this function is `QAT_Quantizer` specific? other quantizers may have a different `fold_bn` function?

Reply:
This function should also work well for other quantizers. (at least for lsq quantizer I think:) ). I will make it a common utility function in the pr that enables batch normalization folding for other quantizers.